core/src/main/kotlin/entity/interaction/ApplicationGuildCommandPermissions.kt

```diff
@@ -0,0 +1,28 @@
+package dev.kord.core.entity.interaction
+
+import dev.kord.common.annotation.KordPreview
+import dev.kord.common.entity.DiscordGuildApplicationCommandPermission
+import dev.kord.common.entity.Snowflake
+import dev.kord.core.cache.data.GuildApplicationCommandPermissionData
+import dev.kord.core.cache.data.GuildApplicationCommandPermissionsData
+
+@KordPreview
+class GuildApplicationCommandPermission(val data: GuildApplicationCommandPermissionData) {
+
+    val id: Snowflake get() = data.id
+
+    val type: DiscordGuildApplicationCommandPermission.Type get() = data.type
+
+    val permission: Boolean get() = data.permission
+}
+
+@KordPreview
+class GuildApplicationCommandPermissions(val data: GuildApplicationCommandPermissionsData) {
+    val id: Snowflake get() = data.id
+
+    val applicationId: Snowflake get() = data.applicationId
+
+    val guildId: Snowflake get() = data.guildId
+
+    val permissions: List<GuildApplicationCommandPermissionData> get() = data.permissions
```

Comment:
This should be the core entity (GuildApplicationCommandPermission) not its core data (GuildApplicationCommandPermissionData)